config.frontend.test.yaml

```diff
@@ -98,6 +100,30 @@ wiktionary_project: &wiktionary_project
                 options: *default_options
           /{api:sys}: *default_sys
 
+commons_project: &commons_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/default.wmf.yaml
+                options: *default_options
+              - path: projects/proxy.yaml
+                options: *proxy_options
+          /{api:sys}: *default_sys
```

Comment:
How's this definition different from the default_project? I think you forgot to bring in recommendation endpoints here.

Reply:
Adding the recommend-caption endpoints in the config file seems is breaking on tests, it seems that the host for appeditortask is ignored.

Reply:
It shouldn't be if you pass the options in there

Reply:
My attempt were:
```
commons_project: &commons_project
  x-modules:
    - spec:
        paths:
          /{api:v1}:
            x-modules:
              - path: projects/v1/default.wmf.yaml
                options: *default_options
              - spec:
                  paths:
                    /data:
                      x-modules:
                        - path: v1/recommend-caption.yaml
                          options: '{{options.appeditortasks}}'
              - path: projects/proxy.yaml
                options: *proxy_options
          /{api:sys}: *default_sys
```
The routes are fine when accessing localhost:7231/commons.wikimedia.org/?spec, but when accessing the endpoint the results got 404 status, That's why I moved to the default project.

Reply:
Should work pretty much exactly how it does for the wikidata descriptions thing.

Reply:
I thought the same, but got different results when testing, am I missing something here?

Reply:
you can
a) put the /data right into the parts in the recommend-caption
b) Do this

```
              - spec:
                  paths:
                    /data:
                      x-modules:
                        - path: v1/recommend-caption.yaml
                          options: '{{options.appeditortasks}}'
                 options: '{{options}}'
```

You're passing `options` on every module load and here you have 2.

Reply:
Done.